src/components/StepList/index.tsx

```diff
@@ -0,0 +1,81 @@
+import styled from '@emotion/styled'
+import React, { ComponentProps, ReactNode } from 'react'
+import Bullet from '../Bullet'
+
+export const Steps = styled.ul`
+  list-style: none;
+  padding-left: ${({ theme }) => theme.space['2']};
+  text-align: left;
+  font-size: 16px;
+`
+
+export const Step = styled('li', {
+  shouldForwardProp: prop => !['disabled'].includes(prop.toString()),
+})<{ disabled: boolean }>`
+  margin-bottom: ${({ theme }) => theme.space['3']};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: ${({ theme, disabled }) =>
+    disabled
+      ? theme.colors.neutral.textDisabled
+      : theme.colors.neutral.textStrong};
+`
+
+const StyledDiv = styled('div', {
+  shouldForwardProp: prop => !['size'].includes(prop.toString()),
+})<{ size: Sizes }>`
+  flex: 1;
+  margin-left: 1rem;
```

Comment:
Are we using rem now ?

Reply:
@matthprost Linked to @chambo-e feedback above as font-size is modified depending on "size" props it avoids having twice the same piece of code